components/multi-platform-controller/production-downstream/kflux-ocp-p01/host-config.yaml

```diff
@@ -630,6 +662,23 @@ data:
   dynamic.linux-largecpu-s390x.allocation-timeout: "1800"
   dynamic.linux-largecpu-s390x.instance-tag: prod-s390x-largecpu
 
+  # Same as linux-largecpu-s390x, but with 320 GB storage
+  dynamic.linux-d320-largecpu-s390x.type: ibmz
+  dynamic.linux-d320-largecpu-s390x.ssh-secret: "internal-prod-ibm-ssh-key"
+  dynamic.linux-d320-largecpu-s390x.secret: "internal-prod-ibm-api-key"
+  dynamic.linux-d320-largecpu-s390x.vpc: "konflux-internal-ocp-art-vpc"
+  dynamic.linux-d320-largecpu-s390x.key: "internal-prod-key"
+  dynamic.linux-d320-largecpu-s390x.subnet: "konflux-internal-ocp-art-vpc-subnet"
+  dynamic.linux-d320-largecpu-s390x.image-id: "r006-7af05e6b-b1cd-4467-8a37-770ce2094e8d"
+  dynamic.linux-d320-largecpu-s390x.region: "us-south-2"
+  dynamic.linux-d320-largecpu-s390x.url: "https://us-south.iaas.cloud.ibm.com/v1"
+  dynamic.linux-d320-largecpu-s390x.profile: "cz2-8x16"
+  dynamic.linux-d320-largecpu-s390x.max-instances: "5"
+  dynamic.linux-d320-largecpu-s390x.private-ip: "true"
+  dynamic.linux-d320-largecpu-s390x.allocation-timeout: "1800"
+  dynamic.linux-d320-largecpu-s390x.instance-tag: prod-s390x-largecpu
```

Comment:
PLease keep tags unique, otherwize the flavours with the same tag will be counted all together.
IN this case it must be like `prod-s390x-d320-largecpu`